versioned_docs/version-24.04/monitoring/metrics.md

```diff
@@ -0,0 +1,176 @@
+
+## Where can I see the metrics for a service in Centreon?
+
+On the **Resource status** page, in the **Details** panel for a service, the **Performance data** section gives you the detailed values for all metrics for this service. The values are that of the last check.
+
+Example: We are monitoring a Centreon poller (for this service, via the [Linux SNMP](https://docs.centreon.com/pp/integrations/plugin-packs/procedures/operatingsystems-linux-snmp/#collected-metrics--status) connector). In the following picture, we can see that the **Memory** service returns 6 metrics: **used**, **free**, **used_prct**, **buffer**, **cached** and **shared**.
+
+![image](../assets/monitoring/metrics.png)
+
+You can also:
+
+* See the evolution of all the metrics for a service in the **Graph** tab of the details panel for the service.
+* Display the last value for a metric in a [dashboard](../alerts-notifications/dashboards.md), using the **Single metric** widget.
+
+## What is the format of a metric?
+
+In all cases, the format of the metrics as returned by the plugins is as follows:
+
+```text
+'name_of_the_metric'= value&unit;warning;critical;min;max
+```
+
+* **value**: the value that is displayed in the details panel of the service is the value returned by the last performed check.
+
+* **warning/critical**: [thresholds](#syntax-of-metric-thresholds) for which the status of the metric will be WARNING or CRITICAL. If no threshold is defined, the status of the metrics will always be OK. The value of these thresholds is defined [in a macro specific to the service, or in its EXTRAOPTIONS macro](#how-do-i-define-a-threshold-for-a-specific-metric).
+
+   > You can choose to define only a CRITICAL threshold and no WARNING threshold.
+
+* **min/max**: This is not the minimum/maximum value seen in the data, but the minimum/maximum possible value that the metric can have. For instance, the maximum CPU usage will always be 100%. These values are defined using the [EXTRAOPTIONS macro](#how-do-i-define-a-threshold-for-a-specific-metric).
+
+   * The min/max values make graphs as relevant as possible: the min and max values will be the minimum/maximum values on the Y or X axis of the graph. If no min or max are defined, the graph will adapt its height to the smallest and biggest seen value. If a graph gathers several metrics, definin the same min and max for all allows you to display them with the same scale.
+   * The min/max values can also be used to correct some default values. For instance, if you are checking the network traffic on an interface, this interface provides its theoretical minimum speed to the plugin, but you can change it manually if the actual speed is limited by another device. This impacts the data exploited by capacity reports in [MBI](../reporting/introduction.md).
+   * Min/max values can also be used to display data as a gauge in [MAP](../graph-views/introduction.md).
+
+### Syntax of metric thresholds
+
```

Comment:
Ici on parle seulement des ranges, mais on peut spécifier le format par défaut et un format "inférieur à" qui est aussi utile:

By default the format is **x** where we are alerted if the value is above x.
It's also possible to alert if the value is below x using the format **x:** .